assignment.js

```diff
@@ -32,6 +32,18 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+// declaring test base number
+let baseNumber = 2
+
+// logic for multiplication of baseNumber
+// the following code does 12 "levels" if you include the zero
+// few people are interested in what baseNumber * 0 computes to
+// to not include zero, comment in line 42 + comment out line 43
+// for (i = 1; i < 13; i++) {
+for (i = 0; i < 12; i++) {
```

Comment:
@nhtaylor great use of a `for` loop here. If you interested in how to solve this using a `while` loop - check out the [approach used here](https://github.com/jsd20200127/hw-01-js-basics-control-flow-loops-solution/blob/master/assignment.js#L38)